client/app/routers/main_router.coffee

```diff
@@ -28,7 +28,8 @@ module.exports = class MainRouter extends Backbone.Router
             intent = event.data
             switch intent.action
                 when 'getToken'
-                    iframeName = document.activeElement.id
+                    elements = document.getElementsByTagName 'iframe'
```

Comment:
This will only work with the first static app to be started.
you can use event.source.location.pathname to get the slug directly instead.